### ISSUE TITLE
Initializing PrefixSet with an iterable does not eliminate longer keys.

### ISSUE THREAD

**Issue by Maimonator** [google/pygtrie#29]
Hi there!
Initializing PrefixSet like so:
```
from pygtrie import PrefixSet

x = PrefixSet(["aaa", "aa"])
print(len(x))
```

outputs `2` even though I'd expect it to print `1`

**Comment by mina86** [comment]
Which version of pygtrie are you using? If you're cloning this repository than you're using an old version. Could you try with the newest version released on PyPI or fetch code from https://github.com/mina86/pygtrie.git repository?

**Comment by Maimonator** [comment]
the version from pypi is an older version.
The version on the git repo you sent is fixed, but with the same efficiency as if I would've done it by myself (loop over and just use the `add` function).
Thanks for the help.

**Comment by mina86** [comment]
Thanks for the report. I'll push a new release to PyPI hopefully later today.

**Comment by mina86** [comment]
Fixed version is now released on PyPI as version 2.3.2.
